docs/about.rst

```diff
@@ -23,6 +23,18 @@ However, model selection is a bit more nuanced than simply picking the "right" o
 
 The **model selection triple** was first described in a 2015 SIGMOD_ paper by Kumar et al. In their paper, which concerns the development of next-generation database systems built to anticipate predictive modeling, the authors cogently express that such systems are badly needed due to the highly experimental nature of machine learning in practice. "Model selection," they explain, "is iterative and exploratory because the space of [model selection triples] is usually infinite, and it is generally impossible for analysts to know a priori which [combination] will yield satisfactory accuracy and/or insights."
 
+
+Who is Yellowbrick for?
+-----------------------
+
+Yellowbrick ``Visualizers`` have multiple use cases:
+
+ - For data scientists, they can help evaluate the stability and predictive value of machine learning models and improves the speed of the experimental workflow.
```

Comment:
For this sentence, "improves" should be "improve".

Reply:
Great catch!